Detectors/ITSMFT/ITS/macros/EVE/DisplayEvents.C

```diff
@@ -0,0 +1,141 @@
+/// \file DisplayEvents.C
+/// \brief Simple macro to display ITS clusters and tracks
+
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+#include <iostream>
+#include <array>
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TEveManager.h>
+#include <TEvePointSet.h>
+#include <TEveTrackPropagator.h>
+#include <TEveTrack.h>
+
+#include "EventVisualisationView/MultiView.h"
+
+#include "ITSBase/GeometryTGeo.h"
+#include "DataFormatsITSMFT/Cluster.h"
+#include "DataFormatsITS/TrackITS.h"
+#endif
+
+extern TEveManager* gEve;
+
+static TTree* gClusTree = nullptr;
+static TTree* gTracTree = nullptr;
+
+static TEveElementList* gEvent = nullptr;
+
+void displayEvent(int entry = 0)
```

Comment:
shouldn't this have the same name as the macro???

Reply:
@sawenzel It could be. But not necessarily.
This macro is not run.  It is *loaded* by the (rootlogon.C).    
Once loaded, it provides several functions that are called by a user from the ROOT command
line, in essentially random order.  That's why...

Reply:
ok

Reply:
@iouribelikov Maybe you are right, in some way.  If we try to add a dummy function with the same name as the macro, the test will launch *that* function. And become "happy" :-)
Should I try ?